lib/rds_sql_server/rds.rb

```diff
@@ -8,17 +8,18 @@
 class Rds
   attr_accessor :client
 
-  REGION             ||= 'us-east-1'
-  AVAILABILITY_ZONE  ||= 'us-east-1a'
-  USERNAME           ||= ENV.fetch('RDS_USERNAME')
-  PASSWORD           ||= ENV.fetch('RDS_PASSWORD')
-  DB_INSTANCE_CLASS  ||= ENV.fetch('RDS_DB_INSTANCE_CLASS')
-  DB_ENGINE          ||= ENV.fetch('RDS_DB_ENGINE')
-  SECURITY_GROUP_IDS ||= [ENV.fetch('RDS_SECURITY_GROUP_ID')].freeze
-  DEFAULT_IDENTIFIER ||= ENV.fetch('RDS_IDENTIFIER') { 'testing' }
-  RDS_KMS_KEY_ID     ||= ENV.fetch('RDS_KMS_KEY_ID')
-  DB_SUBNET_GROUP    ||= ENV.fetch('DB_SUBNET_GROUP') { 'without us-east-1e' }
-  MAX_WAIT_TIME      ||= 1.hour
+  REGION              ||= 'us-east-1'
+  AVAILABILITY_ZONE   ||= 'us-east-1a'
+  USERNAME            ||= ENV.fetch('RDS_USERNAME')
+  PASSWORD            ||= ENV.fetch('RDS_PASSWORD')
+  DB_INSTANCE_CLASS   ||= ENV.fetch('RDS_DB_INSTANCE_CLASS')
+  DB_ENGINE           ||= ENV.fetch('RDS_DB_ENGINE')
+  SECURITY_GROUP_IDS  ||= [ENV.fetch('RDS_SECURITY_GROUP_ID')].freeze
+  DEFAULT_IDENTIFIER  ||= ENV.fetch('RDS_IDENTIFIER') { 'testing' }
+  RDS_KMS_KEY_ID      ||= ENV.fetch('RDS_KMS_KEY_ID')
+  DB_SUBNET_GROUP     ||= ENV.fetch('DB_SUBNET_GROUP') { 'without us-east-1e' }
+  MAX_WAIT_TIME       ||= 1.hour
+  PUBLICLY_ACCESSIBLE ||= ENV.fetch('RDS_PUBLICLY_ACCESSIBLE') { 'false' }.to_s.downcase == 'true'
```

Comment:
What do you think about:
```suggestion
  PUBLICLY_ACCESSIBLE ||= !(Rails.env.development? || Rails.env.test?)
```